### PR TITLE
Update to match updated 1st paragraph

### PR DESCRIPTION
**Pull request by nuritzi**
Updating to match https://github.com/flutter/.github/blob/main/CONTRIBUTING.md per https://github.com/flutter/.github/pull/8

@Hixie adding you here since you helped review the other PR.

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ _tl;dr: join [Discord](https://github.com/flutter/flutter/wiki/Chat), be [courte
 Welcome
 -------
 
-We invite you to join our team! Everyone is welcome to contribute code
-via pull requests, to file issues on GitHub, to help people asking for
-help on our mailing lists, our chat channels, or on Stack Overflow, to
-help triage, reproduce, or fix bugs that people have filed, to add to our
-documentation, or to help out in any other way.
+We invite you to join the Flutter team, which is made up of volunteers and sponsored folk alike!
```
Comment (MrBirb) [inline CONTRIBUTING.md:11]: nit: folks

Comment (nuritzi) [reply CONTRIBUTING.md]: Edited, thanks, @MrBirb!

**Review by goderbauer — APPROVED** [review]
LGTM

**Comment by Hixie** [comment]
This change got made in one of the other sync PRs recently.

**Pull request closed without merging**